protected/views/layouts/new_datasetpage.php

```diff
@@ -4,8 +4,10 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="language" content="en" />
-    <meta name="robots" content="noindex">
-    <meta name="googlebot" content="noindex">
+    <meta name="robots" content="index">
+    <meta name="googlebot" content="index">
```

Comment:
This will need updating based on [comment](https://github.com/gigascience/gigadb-website/issues/515#issuecomment-740445381) by @only1chunts. Also need to repeat for layout classes below.

Reply:
is the googlebot tag actually needed? I think google bots adhere to the "robots" meta tags so unless we want google to treat us differently to other bots (which we dont) then we shouldn't need to explicitly name googlebot.
Please confirm this is correct as I'm no expert in these matters! but I think we should only need:
<meta name="robots" content="index, nofollow"/>

Reply:
Hi @only1chunts , from [here](https://developers.google.com/search/docs/advanced/crawling/special-tags) it says:
```
The <meta name="robots"> tag applies to all search engines, 
while the <meta name="googlebot"> tag is specific to Google.
```
`robots` tag has included Google already, so I think `googlebot` could be removed.